package.json

```diff
@@ -20,7 +20,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "apidoc": "apidoc -o apidoc-out -i src/controllers/",
     "integration": "integration all",
-    "bump": "version-bump"
+    "bump": "version-bump",
+    "drop-oracle-tables": "scripts/drop-oracle-tables.sh"
```

Comment:
So the reason for this script is that you can't pass env variables to to `knex`? Is this why `DYLD_LIBRARY_PATH=/opt/oracle/instantclient knex --knexfile=src/knexfile.js migrate:latest` doesn't work?